tools/internal_ci/linux/gcr_upload.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python2.7
```

Comment:
suggested location: tools/gcp/utils  - the utilities for google compute platform